.github/workflows/ci.yml

```diff
@@ -14,27 +14,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scala-version: [2.11, 2.12, 2.12]
-        spark-version: ["2.4", "3"]
-        neo4j-version: ["3.5", "4.0", "4.1", "4.2", "4.3", "4.4"]
+        java-version: [8, 11, 17]
+        scala-version: [2.12, 2.13]
+        spark-version: ["3"]
+        neo4j-version: ["4.4", "5"]
+        exclude:
+          - spark-version: 3
+            scala-version: 2.11
```

Comment:
do we need this as scala 2.11 is no longer in the list?

Reply:
good catch, thanks :)

Reply:
fixed in 01f4b63e7fd14ed84a604485e4c4ca7c31b9fe24